### ISSUE TITLE
[1.17.1] → [1.12.2] Usage event causes a lag of the character's movement and block placement

### ISSUE THREAD

**Issue by Felix14-v2** [Earthcomputer/multiconnect#217]
[[Demo]](https://youtu.be/r1sp_Xn-BuM)

Server: play.teslacraft.org
Anticheat: possibly AAC

It seems that the logs do not reflect the problem.

Also, thank you so much for your great work!

**Comment by Earthcomputer** [comment]
Can't reproduce, it must be something weird that that server is doing. Does it also happen if you connect with vanilla 1.12?

**Comment by Felix14-v2** [comment]
> Does it also happen if you connect with vanilla 1.12?
 No, vanilla server is good. It seems that this issue caused by server's anticheat.

**Comment by RaphiMC** [comment]
1.17 clients send an additional movement packet when the player interacts with an item. So the client sends one movement packet more per interaction (which block places also are I think) and that causes anticheats to flag you

**Comment by Felix14-v2** [comment]
@Earthcomputer , can this issue be reopened? Nevertheless, the goal of multiconnect is to ensure compatibility of the client and the server in order to avoid problems with anticheats, but there is clearly a problem with sending unnecessary data to the server.

**Comment by Earthcomputer** [comment]
Yeah now RaphiMC has said what the cause is it should be feasible to fix

**Comment by Earthcomputer** [comment]
Is this fixed as of the latest release?

**Comment by Felix14-v2** [comment]
Absolutely right, thank you very much!

**Comment by Felix14-v2** [comment]
@Earthcomputer , seems that this bug has returned, alongside with the weird missing textures:

https://user-images.githubusercontent.com/75726196/226054712-305872f8-b96e-4c25-a258-50dea8860a13.mp4


https://user-images.githubusercontent.com/75726196/226056181-5c83787c-41ff-4207-b44d-fd6744d7aeed.mp4

[latest.log](https://github.com/Earthcomputer/multiconnect/files/11006396/latest.log)
